### PR TITLE
feat(scoop-status): Add flag to disable remote checking

### LINKED ISSUE #5044: [Bug] Optionally disable remote-checking in `scoop status`
#### Current Behavior
scoop status takes ~3 times more time in v0.2.3 vs v.0.2.2

#### Additional context/output
scoop status in v0.2.3
image

scoop status in v0.2.2
image

top functions
image

#### Possible Solution
It seems that in 0.2.3 Test-UpdateStatus is run for each bucket which is quite expensive.
image

### System details
windows 11 (21H2 build 22000.778)
x64
ps core: 7.2.5
git version 2.37.0.windows.1

#### Scoop Configuration

```json
{
  "lastupdate": "2022-07-12T16:41:04.1824139+02:00",
  "SCOOP_REPO": "https://github.com/lukesampson/scoop",
  "SCOOP_BRANCH": "master",
  "shim": "kiennq",
  "aria2-warning-enabled": false
}
```

### PR DESCRIPTION
**Pull request by rashil2000**
#### Description

The improved status command checks for all outdated buckets. This can lead to slowdowns, so this optional flag disables all remote checking.

#### Motivation and Context

Closes #5044

#### How Has This Been Tested?

image

#### Checklist:

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I have ensured that I am targeting the `develop` branch.
- [x] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.
- [x] I have added an entry in the CHANGELOG.

### CODE REVIEW

**Comment by niheaven** [comment]
Add a `break` and save half time, this may be another choice.

image

image

**Comment by rashil2000** [comment]
> Add a break and save half time, this may be another choice.

Nice idea! 
Added.

File: libexec/scoop-status.ps1 [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Usage: scoop status
 # Summary: Show status and check for new app versions
+# Help: Options:
+#   -r, --no-remotes      Disables remote fetching/checking for Scoop and buckets
```
Comment (niheaven) [inline libexec/scoop-status.ps1:4]: `-l`, `--local`?

Comment (rashil2000) [reply libexec/scoop-status.ps1:4]: Will change it

**Pull request merged**